Tensile/KernelWriter.py

```diff
@@ -945,34 +959,42 @@ def kernelBody( self, kernel, tensorParametersA, tensorParametersB ):
         # can't do shadow initC with multiple summation since this resets the ValuC counters
         # on each unroll iteration.
         self.doShadowInit = 1 # 1 is just store setup
+
+    # in HPA mode, can do the alpha, beta conversion f16->f32 only once
+    kl.append( self.checkAlphaBetaForHPA(kernel))
```

Comment:
any reason to move this code block earlier

Reply:
This is for HPA issue, originally, the conversion from f16 to f32 is done in each tile loop, and stores the f32 value back to alpha / beta everytime, if we keep this conversion inside the pk-loop, then we need to test if this is the first time doing conversion. (if not the first time, then skip the conversion). 

And also, in the original version, this conversion code is scattered all over the different case (OptNNL, non-OptNNL, SRVW, Non-SRVW) which makes the maintenance difficult. 

Put this conversion before the PK-loop so that we can do it only once.

Reply:
Done, I've made the change, now this is done earlier only when we have PK